node/DEVELOPER.md

```diff
@@ -13,6 +13,7 @@ The GLIDE Node wrapper consists of both TypeScript and Rust code. Rust bindings
 Software Dependencies
 
 > Note: Currently, we only support npm major version 8. f you have a later version installed, you can downgrade it with `npm i -g npm@8`.
+> Note: Currently, we only support Node version 16 and above, if you have previous version you can update it using NVM - https://github.com/nvm-sh/nvm?tab=readme-ov-file#install--update-script .
```

Comment:
You know what, the supported node version is written in the README file, so lets not write again that the supported version is 16 and above, since then we'll need to maintain it in two places. Instead, you can share a link to the supported version, something like:
-"If your NodeJS version is below the supported version specified in the client's documentation (link), you can upgrade it using..."